doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+::
+
+  unique_values = np.unique(a)
+  print(unique_values)
+
+**Output:**
+
+::
+
+  [11 12 13 14 15 16 17 18 19 20]
+
+To get the indices of unique values in a NumPy array (an array of first index positions of unique values in the array), just pass the ``return_index`` argument in ``np.unique()`` as well as your array.
+
+**Input:**
+
+::
+
+  indices_list = np.unique(a, return_index=True)
+  print(indices_list)
+
+**Output:**
+
+::
+
+  [ 0  2  3  4  5  6  7 12 13 14]
+
+You can pass the ``return_counts`` argument in ``np.unique()`` along with your array to get the frequency count of unique values in a NumPy array.
+
+**Input:**
+
+::
+
+  unique_values, occurrence_count = np.unique(a, return_counts=True)
+  print(occurrence_count)
+
+**Output:**
+
+::
+
+  [3 2 2 2 1 1 1 1 1 1]
+
+This also works with 2D arrays. If you start with this array:
+
+::
+
+  a2D = np.array([[1, 2, 3, 4] ,[5, 6, 7, 8] , [9, 10, 11, 12],  [1, 2, 3, 4]])
+
+You can find unique values with:
+
+**Input:**
+
+::
+
+  unique_values = np.unique(a2D)
+  print(unique_values)
+
+**Output:**
+
+::
+
+  [ 1  2  3  4  5  6  7  8  9 10 11 12]
+
+If the axis argument isn't passed, your 2D array will be flattened. 
+
+To get the unique rows or columns, make sure to pass the ``axis`` argument. To find the unique rows, specify ``axis=0`` and for columns, specify ``axis=1``.
+
+**Input:**
+
+::
+
+  unique_rows = np.unique(a2D, axis=0)
+  print(unique_rows)
+
+**Output:**
+
+::
+
+  [[ 1  2  3  4]
+   [ 5  6  7  8]
+   [ 9 10 11 12]]
+
+To get the unique rows, occurrence count, and index position, you can use:
+
+**Input:**
+
+::
+
+  unique_rows, occurence_count, indices = np.unique(a2D, axis=0, return_counts=True, return_index=True)
+  print('Unique Rows: ', '\n', unique_rows) 
+  print('Occurrence Count:', '\n', occurence_count)
+  print('Indices: ', '\n', indices)
+
+**Output:**
+
+::
+
+  Unique Rows:  
+   [[ 1  2  3  4]
+   [ 5  6  7  8]
+   [ 9 10 11 12]]
+  Occurrence Count: 
+   [0 1 2]
+  Indices:  
+   [2 1 1]
+
+`Learn more about finding the unique elements in an array here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.unique.html>`_.
+
+
+Transposing and reshaping a matrix
+----------------------------------
+
+
+::
+
+  arr.reshape()
+  arr.transpose()
+  arr.T()
+
+-----
+
+It's common to need to rotate your matrices. NumPy arrays have the property ``T`` that allows you to transpose a matrix.
+
+.. image:: images/np_transposing_reshaping.png
+
+You may also need to switch the dimensions of a matrix. This can happen when, for example, you have a model that expects a certain input shape that is different from your dataset. This is where the ``reshape`` method can be useful. You simply need to pass in the new dimensions that you want for the matrix.
+
+::
+
+  data.reshape(2,3)
+  data.reshape(3,2)
+
+.. image:: images/np_reshape.png
+
+You can also use ``np.transpose`` to reverse or change the axes of an array according to the values you specify.
```

Comment:
`.transpose` (talking about the method, not the function)

Reply:
Done!